gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemNewIntegrationTest.java

```diff
@@ -304,7 +331,35 @@ public void getDirInfo_sequential() throws Exception {
   }
 
   @Test
-  public void getDirInfo_parallel() throws Exception {
+  public void getDirInfo_objectName_parallel() throws Exception {
```

Comment:
nit: do we want to re-order tests ?

Reply:
I order them logically - from "simpler" `../name` test to more complex `../name/` test, but I can reorder them alphabetically.